src/Couchbase.Lite.Shared/API/Sync/ReplicatorProgress.cs

```diff
@@ -71,7 +71,7 @@ public struct ReplicatedDocument
         [CanBeNull]
         public CouchbaseException Error { get; }
 
-        internal CouchbaseLiteException Exception { get; set; }
+        public CouchbaseLiteException Exception { get; }
```

Comment:
I think I need to be clearer here.  Get rid of this property.  Everything should be on the `Error` property only.